keras/backend/mxnet_backend.py

```diff
@@ -4,6 +4,7 @@
 import warnings
 import mxnet as mx
 import numpy as np
+import keras
```

Comment:
why are we importing the entire library? Please import only the specific modules that you have used in the PR